packages/graphql-serve/src/runtime.ts

```diff
@@ -25,19 +33,48 @@ export const createMongoDBClient = async (): Promise<MongoClient> => {
   return client
 }
 
+export function getConflictMapForModels(model: GraphQLSchema, conflictResolutionStrategy: ConflictResolutionStrategy, deltaTTL: number): DataSyncModelConfigMap {
+  const map: DataSyncModelConfigMap = {};
+
+  const dataSyncModelTypes = filterModelTypes(model).filter(isDataSyncType);
+
+  dataSyncModelTypes.forEach((type: GraphQLObjectType) => {
+    map[type.name] = {
+      enabled: true,
+      deltaTTL: deltaTTL,
+      conflictResolution: conflictResolutionStrategy
+    }
+  })
+
+  return map
+}
+
 /**
  * Method used to create runtime schema
  * It will be part of the integration tests
  */
-export const createRuntime = async (modelDir: string, db: Db, datasync: boolean): Promise<GraphbackAPI> => {
+export const createRuntime = async (modelDir: string, db: Db, datasyncServeConfig: DataSyncServeConfig): Promise<GraphbackAPI> => {
   const model = await loadModel(modelDir);
 
   let graphbackAPI;
-  if (datasync) {
-    graphbackAPI = buildGraphbackAPI(model, {
-      serviceCreator: createDataSyncCRUDService({ pubSub: new PubSub() }),
-      dataProviderCreator: createDataSyncMongoDbProvider(db),
-      plugins: [new DataSyncPlugin()]
+  if (datasyncServeConfig.datasync) {
+    let conflictResolutionStrategy = ClientSideWins;
+    let deltaTTL = 172800;
+    if (datasyncServeConfig.conflict === "client") {
+      conflictResolutionStrategy = ClientSideWins;
+    }
+    if (datasyncServeConfig.conflict === "server") {
```

Comment:
Undocumented `server` value

Reply:
server is supposed to mean serverSideWins, should I expand "server" to "serverSideWins"? Not sure I understand

Reply:
Do you mean that I should describe both options:
https://github.com/aerogear/graphback/blob/f5f02b58740dcdb6d7e13943b80dc3c6b6c3a472/packages/graphql-serve/src/commands/serve.ts#L30